samples/brokers/onvif-video-broker/Program.cs

```diff
@@ -102,7 +102,7 @@ static void Main(string[] args)
 
 			FrameTask = Task.Run(() => Process(RtspUrl));
 
-			var metricServer = new KestrelMetricServer(port: 8000);
+			var metricServer = new KestrelMetricServer(port: 8080);

```

Comment:
why change this?

Reply:
To match the port we are using for the [controller and agent](https://github.com/deislabs/akri/blob/main/docs/prometheus.md#enabling-prometheus-in-akri) and udev broker.

Reply:
ok ... so this is an intentional bug fix :)